moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -911,17 +911,11 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(robot_trajectory::RobotT
                 max_acceleration_scaling_factor, acceleration_scaling_factor);
   }
 
-  // This lib does not actually work properly when angles wrap around, so we need to unwind the path first
-  trajectory.unwind();
```

Comment:
Did you actually test the code with continuous joints and a trajectory that wraps angles?
Perusing the commits I can't see any explanation related to removing this... :cat2:

Reply:
It was just moved down to L1063  :+1: